Sources/JXKit/JXValue.swift

```diff
@@ -454,14 +503,72 @@ extension JXValue {
         }
     }
 
-    /// Returns the JavaScript object as dictionary.
-    @inlinable public var dictionary: [String: JXValue]? {
+    /// Returns the JavaScript object as a dictionary.
+    @inlinable public var dictionary: [String: JXValue] {
         get throws {
-            !isObject ? nil : try self.properties.reduce(into: [:]) { $0[$1] = try self[$1] }
```

Comment:
Made this throw instead of return an optional, as that seems more consistent with other methods.